studio/src/components/layout/sidenav.tsx

```diff
@@ -100,7 +100,7 @@ const Organizations = () => {
             );
             if (currentOrg) {
               router.replace(
-                currentPage === "graph"
+                currentPage === "graph" || currentPage === "invitations"
```

Comment:
Can we generalize more to avoid specifying pages manually?

Reply:
Done